cypress/integration/j5.js

```diff
@@ -0,0 +1,30 @@
+import T3Page from '../obj/tasksPage';
```

Comment:
why is file name `j5.js`, also as a general note i would suggest using names that reflect what kind of test i would find inside file, like `login.js` or `task.js`